packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/rights/vsphere-user-rights.controller.js

```diff
@@ -13,8 +15,18 @@ export default class {
     };
 
     return this.DedicatedCloud.getUserDetail(this.productId, this.userId)
-      .then((details) => {
-        this.selectedUser = details;
+      .then((user) => {
+        set(user, 'loginUsername', user.login.split('@')[0]);
+        set(user, 'loginDomain', user.login.split('@')[1]);
```

Comment:
**suggestion:**
```suggestion
        const [ userName, domain ] = user.login.split('@');
        set(user, 'loginUsername', userName);
        set(user, 'loginDomain', domain);
```

Reply:
Sure, will do that on both controllers, as I copied this code from an existing controller :)

Reply:
Done 👍🏻